nextflow_schema.json

```
@@ -126,40 +125,42 @@
-                    "fa_icon": "fas fa-align-right"
+                    "help_text": "Specifies the minimum coverage percent of the annotated genome. This must be set between 0-100.\n\nFor more information. please check PROKKA [documentation](https://github.com/tseemann/prokka).\n\n> Modifies tool parameter(s):\n> - PROKKA: `--coverage`",
+                    "fa_icon": "fas fa-align-right",
+                    "minimum": 0,
+                    "maximum": 100
                 },
                 "annotation_prokka_cdsrnaolap": {
                     "type": "boolean",
-                    "description": "Allow transfer RNA (trRNA) to overlap coding sequences (CDS). Transfer RNAs are short stretches of nucleotide sequences that link mRNA and the amino acid sequence of proteins. Their presence helps in the annotation of the sequences, because each trRNA can only be attached to one type of amino acid.",
-                    "help_text": "Allow transfer RNA (trRNA) to overlap coding sequences (CDS).\n\n> Modifies tool parameter(s):\n> - PROKKA: `--cdsrnaolap`",
+                    "description": "Allow transfer RNA (trRNA) to overlap coding sequences (CDS).",
+                    "help_text": "Allow transfer RNA (trRNA) to overlap coding sequences (CDS). Transfer RNAs are short stretches of nucleotide sequences that link mRNA and the amino acid sequence of proteins. Their presence helps in the annotation of the sequences, because each trRNA can only be attached to one type of amino acid.\n\nFor more information. please check PROKKA [documentation](https://github.com/tseemann/prokka).\n\n> Modifies tool parameter(s):\n> - PROKKA: `--cdsrnaolap`",
                     "fa_icon": "fas fa-align-justify"
                 },
                 "annotation_prokka_rnammer": {
                     "type": "boolean",
                     "description": "Use RNAmmer for rRNA prediction.",
-                    "help_text": "Activates RNAmmer instead of the PROKKA default Barnap for rRNA prediction during the annotation process. RNAmmer classifies ribosomal RNA genes in genome sequences by using two levels of Hidden Markov Models. Barnap uses the nhmmer tool that includes HMMER 3.1 for HMM searching in RNA:DNA style. More info about 'Barnap' can be found [here](https://github.com/tseemann/barrnap).\n\n> Modifies tool parameter(s):\n> - PROKKA: `--rnammer`",
+                    "help_text": "Activates RNAmmer instead of the PROKKA default Barnap for rRNA prediction during the annotation process. RNAmmer classifies ribosomal RNA genes in genome sequences by using two levels of Hidden Markov Models. Barnap uses the nhmmer tool that includes HMMER 3.1 for HMM searching in RNA:DNA style. More information about 'Barnap' can be found [here](https://github.com/tseemann/barrnap).\n\nFor more information. please check PROKKA [documentation](https://github.com/tseemann/prokka).\n\n> Modifies tool parameter(s):\n> - PROKKA: `--rnammer`",
```

I did not like the wo times `more information` in a row ...
```suggestion
                    "help_text": "Activates [RNAmmer](https://services.healthtech.dtu.dk/service.php?RNAmmer-1.2) instead of the PROKKA default [Barrnap](https://github.com/tseemann/barrnap) for rRNA prediction during the annotation process. RNAmmer classifies ribosomal RNA genes in genome sequences by using two levels of Hidden Markov Models. Barrnap uses the nhmmer tool that includes HMMER 3.1 for HMM searching in RNA:DNA style.\n\nFor more information please check PROKKA [documentation](https://github.com/tseemann/prokka).\n\n> Modifies tool parameter(s):\n> - PROKKA: `--rnammer`",
```